views/tables/ChartsTable/index.jsx

```diff
@@ -137,7 +137,9 @@ const ChartsTable = ({
           </Link>
         )
       case 'updatedAt':
-        return chart[property] ? dayjs(chart[property]).format(DATE_FORMAT) : ''
+        return chart[property]
+          ? dayjs(chart[property]).format(DATE_FORMAT)
+          : dayjs(chart['createdAt']).format(DATE_FORMAT)
```

Comment:
if we're defaulting the updatedAt property to the same value as the createdAt then we shouldn't need to do this.

Reply:
[9cf0ba5](https://github.com/AMP-SCZ/dpdash/pull/704/commits/9cf0ba5e061b38ea21f2553f55d80932166c2293)